app/templates/dashboard.html

```diff
@@ -19,8 +19,10 @@
         </div>
         <div class="details">
             <h2>{{ current_user.firstName }} {{ current_user.lastName }}</h2>
-            <p>Unique ID: {{ current_user.knewbie_id }} </p>
-            <a href="#" class="btn">Progress Report</a>
+            {% if current_user.is_authenticated and current_user.urole == 'student' %}
```

Comment:
If the dashboard requires login then there's no need to check `is_authenticated` correct?